CONTRIBUTING.md

```diff
@@ -32,6 +32,15 @@ Make the development version available globally:
 $ pip install -e .
 ```
 
+Install the `black` formatter hook with:
+```sh
+pre-commit install
+```
+
+With help of pre-commit, your future commits will be reformatted with our black configuration
+which includes settings like `skip-string-normalization = true` and `max-line-length = 79`.
+If you like to learn more please check [black](https://github.com/python/black).
```

Comment:
Can you change it to "If you would like to learn more", slightly better grammatically. Otherwise looks ready to merge